library/core/src/iter/sources/repeat.rs

```diff
@@ -98,7 +98,7 @@ impl<A: Clone> Iterator for Repeat<A> {
     }
 
     fn last(self) -> Option<A> {
```

Comment:
`#[track_caller]` should be added back here.

Reply:
Done!